delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/projects/Project.scala

```diff
@@ -57,4 +73,7 @@ object Project {
   implicit val projectJsonLdEncoder: JsonLdEncoder[Project] =
     JsonLdEncoder.fromCirce(context)
 
+  implicit val metadataEncoder: Encoder.AsObject[Metadata]    = deriveConfiguredEncoder[Metadata]
```

Comment:
I think this one should be private and only the `JsonLdEncoder[Metadata]` public